test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -66,17 +67,29 @@ import (
 	rbacv1 "k8s.io/api/rbac/v1"
 )
 
-// Test_ValidateNamespace_NoParams tests a ValidatingAdmissionPolicy that validates creation of a Namespace with no params.
-func Test_ValidateNamespace_NoParams(t *testing.T) {
-	forbiddenReason := metav1.StatusReasonForbidden
+// Test_ValidateNamespace_NoParams_Success tests a ValidatingAdmissionPolicy that validates creation of a Namespace
+// with no params via happy path.
+func Test_ValidateNamespace_NoParams_Success(t *testing.T) {
+	generic.PolicyRefreshInterval = 10 * time.Millisecond
```

Comment:
A note here to clarify that this is a stop-gap until we can reduce polling interval by default?  (or maybe set a const shared by all tests set to 10ms and put the comment on that?)

Reply:
Done